commit 0bec5ffa7923e005ae51da8a78f26caad9376b48
Author: Felix Fietkau <nbd@nbd.name>
Date:   Wed Jan 11 16:32:13 2017 +0200

    ath5k: drop bogus warning on drv_set_key with unsupported cipher
    
    commit a70e1d6fd6b5e1a81fa6171600942bee34f5128f upstream.
    
    Simply return -EOPNOTSUPP instead.
    
    Signed-off-by: Felix Fietkau <nbd@nbd.name>
    Signed-off-by: Kalle Valo <kvalo@qca.qualcomm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1287_0bec5ffa7923e005ae51da8a78f26caad9376b48.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1287_0bec5ffa7923e005ae51da8a78f26caad9376b48.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
